app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -28,7 +28,7 @@
         android:id="@+id/add_profile_activity_scroll_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/add_profile_background">
+        android:background="@color/oppia_add_profile_background_color">
```

Comment:
A first thought: we probably don't need to prefix *everything* with 'oppia_'. I can see it being useful for custom colors (i.e. in color_defs), but everywhere else it's redundant.

Reply:
Done.